display.js

```diff
@@ -883,14 +883,16 @@ RecipeTable.prototype = {
 }
 
 // Re-renders the current solution, without re-computing it.
-function display() {
+function display(dontChangeHash) {
     // Update the display of the target rate text boxes, if needed.
     for (var i = 0; i < build_targets.length; i++) {
         build_targets[i].getRate()
     }
     var totals = globalTotals
 
-    window.location.hash = "#" + formatSettings()
+    if (!dontChangeHash) {
```

Comment:
I don't like flags which are double-negatives like this. I would prefer a name which is affirmative, like "keepHash".